fvm/state/accounts.go

```diff
@@ -335,9 +350,14 @@ func (a *Accounts) TouchContract(contractName string, address flow.Address) {
 		panic(err)
 	}
 	if contractNames.Has(contractName) {
-		a.ledger.Touch(string(address.Bytes()),
+		// TODO change me to touch
```

Comment:
Does this _need_ to be a touch or does it still work as a read?

Reply:
updated